### ISSUE TITLE
Web-Proxy does not work on localhost, but fine from the container, other containers and machines in network

### ISSUE THREAD

**Issue by GAS85** [haugene/docker-transmission-openvpn#2179]
### Is there a pinned issue for this?

- [X] I have read the pinned issues and could not find my issue

### Is there an existing or similar issue/discussion for this?

- [X] I have searched the existing issues
- [X] I have searched the existing discussions

### Is there any comment in the documentation for this?

- [X] I have read the documentation, especially the FAQ and Troubleshooting parts

### Is this related to a provider?

- [X] I have checked the provider repo for issues
- [X] My issue is NOT related to a provider

### Are you using the latest release?

- [X] I am using the latest release

### Have you tried using the dev branch latest?

- [X] I have tried using dev branch

### Config used

```yaml
version: "3.6"
services:  
  transmission-vpn:
    container_name: transmission-vpn
    image: haugene/transmission-openvpn
    cap_add:
      - NET_ADMIN
    devices:
      - /dev/net/tun
    restart: always
    ports:
      # RPC
      - "9091:9091"
      # Web Proxy
      - "9092:8888"
    dns:
      - 84.200.69.80
      - 84.200.70.40
    volumes:
      - ${WORKINGDIR}/docker/transmission-vpn:/data
      - ${WORKINGDIR}/docker/transmission-vpn/scripts:/scripts
      - ${WORKINGDIR}/Downloads:/data/watch
      - ${WORKINGDIR}/Downloads/completed:/data/completed
      - ${WORKINGDIR}/Downloads/incomplete:/data/incomplete
    logging:
      driver: json-file
      options:
          max-size: 10m
    environment:
      ...
      - LOCAL_NETWORK=192.168.0.0/24
      - PUID=${PUID}
      - PGID=${PGID}
      - TZ=${TZ}
      - WEBPROXY_ENABLED=true
      - WEBPROXY_PORT=8888
      - WEBPROXY_USERNAME=<some user name>
      - WEBPROXY_PASSWORD=<some password>
```

### Current Behavior

I could use web proxy via container:

```bash
docker exec transmission-vpn curl -x "http://user:password@localhost:8888" "https://www.google.com"
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0<!doctype html><html itemscope="" itemtype="http://schema.org/WebPage" lang="de"><head><meta content="text/html; charset=UTF-8" http-equiv="Content-Type"><meta content="/images/branding/googleg/1x/googleg_standard_color_128dp.png" itemprop="image"><title>Google</title>
...
```

Also from other machines working fine:

```bash
curl -x "http://user:password@ServerIpOrHostname:9092" "https://www.google.com"
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0<!doctype html><html itemscope="" itemtype="http://schema.org/WebPage" lang="de"><head><meta content="text/html; charset=UTF-8" http-equiv="Content-Type"><meta content="/images/branding/googleg/1x/googleg_standard_color_128dp.png" itemprop="image"><title>Google</title>
...
```

~~But [config.privoxy.org](http://config.privoxy.org/) shows that I do not use proxy (as per https://haugene.github.io/docker-transmission-openvpn/web-proxy/), IP address check shows that traffic goes via Proxy.~~ It shows Proxy config UI, everything fine here.

But from the `localhost` on Docker Host machine.

```bash
curl -x "http://user:password@localhost:9092" "https://www.google.com"
% Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:--  0:00:01 --:--:--     0
  0     0    0     0    0     0      0      0 --:--:--  0:00:02 --:--:--     0
  0     0    0     0    0     0      0      0 --:--:--  0:00:03 --:--:--     0
  0     0    0     0    0     0      0      0 --:--:--  0:00:04 --:--:--     0
  0     0    0     0    0     0      0      0 --:--:--  0:00:05 --:--:--     0
  0     0    0     0    0     0      0      0 --:--:--  0:00:06 --:--:--     0
^C
```

### Expected Behavior

I really have no ideas why 1 port is not behaving similar to other ports. There is no special configuration about it.
`iptables` enable access to the ports range `9000-9100` via TCP and should not be the problem. Port `9091` accessible from the `localhost` without any problems but `9092` not.

### How have you tried to solve the problem?

1) Different calls via Curl or browser
2) Proxy without auth

### Log output

Only this was related to Proxy in logs:
```
Privoxy: Starting
Privoxy: Using config file at /etc/privoxy/config
Privoxy: Setting port to 8888
Privoxy: Running as PID 108
```

### Environment

```markdown
- OS: Ubuntu 20.04
- Docker: 20.10.12, build e91ed57
```

**Comment by edgd1er** [comment]
some ideas:

* check if port 9092 is listened on all interfaces: `ss -l src :9092`. you could add address in ports section: `0.0.0.0:9092:8888`
* sometimes, localhost is resolved as 127.0.1.1, your container may not listen on that address. replace localhost with an ip address.
* The term "from the localhost" could from the host (ubuntu) or from the container. if it is from within the container, then the port is 8888.

**Comment by edgd1er** [comment]
The last thing I could think of is, specific rules for privoxy.
the logfile /var/log/privoxy/logfile may have some information about denied access coming from a specific network range.
